env/ORION.env

```diff
@@ -17,7 +17,7 @@ step=$1
 
 export npe_node_max=40
 export launcher="srun -l --export=ALL"
-export mpmd_opt="--multi-prog"
+export mpmd_opt="--multi-prog --output=${step}.%J.%t.out"
```

Comment:
Are the changes to this file relevant to this PR?

Reply:
I asked a similar question while discussing this PR offline with Walter. I recommend pulling this `--output` change out and submitting in a separate PR, in addition to setting in the other env files. It's useful to have.

Reply:
This has been pulled out into PR #1469 and extended to other slurm machines.